src/main/java/uk/gov/hmcts/darts/dailylist/controller/DailyListController.java

```diff
@@ -46,32 +51,83 @@ public class DailyListController implements DailyListsApi {
     @Autowired
     private DailyListProcessor processor;
 
+    @Override
```

Comment:
Can we remove these annotations and leave them in the supper class. it makes looking at the rest controller hard than it needs to be.

Reply:
done